photoeditor/src/main/java/ja/burhanrashid52/photoeditor/TextStyleBuilder.java

```diff
@@ -32,6 +32,24 @@ public void withTextSize(@NonNull float size) {
         values.put(TextStyle.SIZE, size);
     }
 
+    /**
+     * Set this textShadow style
+     *
+     * @param radius Radius of the shadow to apply on text
+     * @param dx Horizontal distance of the shadow
+     * @param dy Vertical distance of the shadow
+     * @param color Color of the shadow
+     */
+    public void withTextShadow(@NonNull float radius, float dx, float dy, int color) {
```

Comment:
Please move these 4 parameter to `TextShadow` object and put them in map with `TextStyle.SHADOW` key ?
`TextShadow` does not need to have setters they can be created using constructor only.

Reply:
Just did it, committing now. Hope I got it right in Java.

Also used the commit to do some little bumping in dependencies versions.